plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/SystemProperties.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kudu;
+
+import java.util.Objects;
+
+import static com.google.common.base.Preconditions.checkState;
+
+public final class SystemProperties
```

Comment:
nit: KerberosAuthenticator and LoginBasedSubjectProvider should now re-use this. They already have a copy of what's here.

follow-up commit.

Reply:
Did you mean follow up commit within this PR/or as a separate PR?

Reply:
Either works for me - it's a related change after all.

Reply:
Did this, however `LoginBasedSubjectProvider` lives in the trino client module and I do not think introducing a new dependency in the trino client module is worth the small reduction in duplicate code.

Reply:
Fair point.